diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/ASTNodeUtilsTest.kt

```diff
@@ -266,26 +290,42 @@ class ASTNodeUtilsTest {
             }
         """.trimIndent()
         applyToCode(code){node ->
-            val list = mutableListOf<IElementType>()
+            val setSibling = mutableSetOf<IElementType>()
+            val setParent = mutableSetOf<IElementType>()
+
+            if (node.treeParent != null) {
+                node.treeParent.getChildren(null).forEach {
+                    setParent.add(it.elementType)
+                }
+            }
+            else
+                setParent.add(node.elementType)
+            node.allSiblings(false).forEach {
+                setSibling.add(it.elementType)
+            }
             node.allSiblings(true).forEach {
-                list.add(it.elementType)
+                setSibling.add(it.elementType)
             }
```

Comment:
You call `allSiblings(false)` and right after, without checking, `allSiblings(true)` - what is the intention of this test? I suggest to refactor it a little. Firstly, why use element types when you can compare collections of nodes directly? Secondly, I suggest to use more functional style: instead of creating mutable collections and populating them inside loops, you can use koltin collections API calls like `map` or `toSet()` if you need to create sets.